countryspec.go

```diff
@@ -0,0 +1,16 @@
+package stripe
+
+// CountrySpec is the resource representing the rules required for a Stripe account.
+// For more details see https://stripe.com/docs/api/#country_specs.
+type CountrySpec struct {
+	ID                             string                         `json:"id"`
+	SupportedBankAccountCurrencies map[string][]string            `json:"supported_bank_account_currencies"`
+	SupportedPaymentCurrencies     []string                       `json:"supported_payment_currencies"`
+	SupportedPaymentMethods        []string                       `json:"supported_payment_methods"`
+	VerificationFields             map[string]map[string][]string `json:"verification_fields"`
```

Comment:
Not a big one, but looking at some other models like `Charge` ([see here](https://github.com/stripe/stripe-go/blob/e4ca8d9583a2be58c844e890d2dbf85457aaace4/charge.go#L58,L84)), it seems like there's a slight preference here for defining new types instead of using long untyped constructs (e.g. `map[string]map[string][]string`). This is a little more convenient for those who have auto-complete systems set up their editors because the structure becomes self-describing.

So for example we might consider even:

``` go
map[Currency][]string
```

Or:

``` go
map[AccountType]VerificationFields
```

With something like:

```
type AccountType string

type VerificationFields struct {
    AdditionalFields []string
    MinimumFields []string
}
```

Thoughts?